<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>459</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>858</width>
    <height>459</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>858</width>
    <height>459</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="txiUsername">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>203</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入用户名</string>
   </property>
   <property name="statusTip">
    <string>请输入用户名</string>
   </property>
   <property name="whatsThis">
    <string>请输入用户名</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txiPassword">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>238</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入密码</string>
   </property>
   <property name="statusTip">
    <string>请输入密码</string>
   </property>
   <property name="whatsThis">
    <string>请输入密码</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>76</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>点击按钮登录系统</string>
   </property>
   <property name="statusTip">
    <string>点击按钮登录系统</string>
   </property>
   <property name="whatsThis">
    <string>点击按钮登录系统</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/btn/images/login_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginCancle">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>320</y>
     <width>76</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>点击按钮清空用户名和密码</string>
   </property>
   <property name="statusTip">
    <string>点击按钮清空用户名和密码</string>
   </property>
   <property name="whatsThis">
    <string>点击按钮清空用户名和密码</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/btn/images/cancle_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txiMAC">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>275</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>请输入密码</string>
   </property>
   <property name="statusTip">
    <string>请输入密码</string>
   </property>
   <property name="whatsThis">
    <string>请输入密码</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>新用户注册</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>记住密码？</string>
   </property>
   <widget class="QRadioButton" name="rdo_save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdo_unsave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>否</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="txtTitle">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg/images/bg/transparent.png);
color: rgb(255, 255, 255);
font: 24pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>飞速科技</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="txt_version">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txiUsername</tabstop>
  <tabstop>txiPassword</tabstop>
  <tabstop>txiMAC</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnLoginCancle</tabstop>
  <tabstop>btnRegister</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnLogin</sender>
   <signal>clicked()</signal>
   <receiver>LoginForm</receiver>
   <slot>logging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoginCancle</sender>
   <signal>clicked()</signal>
   <receiver>LoginForm</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRegister</sender>
   <signal>clicked()</signal>
   <receiver>LoginForm</receiver>
   <slot>showRegisterBox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logging()</slot>
  <slot>reset()</slot>
  <slot>showRegisterBox()</slot>
 </slots>
</ui>
